app/views/organisations/_what_we_do.html.erb

```diff
@@ -27,7 +27,9 @@
       <% end %>
     </div>
 
-    <% if @what_we_do.has_share_links? %>
+    <% social = t_social_media_accounts(@what_we_do.social_media_links, @_request) %>
```

Comment:
It feels like this assignment probably doesn't belong in the view.  Would it be possible to remove this line from the view and do the translations in the presenter (a bit like the Contacts Presenter), so the `share_links` method returns the translated links only?